resources/js/views/Teams/Teams.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+    <section>
+        <div class="columns">
+
+            <div class="column is-one-fifth teams-left-col">
+                <p class="teams-title">OpenLitterMap Teams</p>
+
+                <div v-for="i in items" class="team-flex" @click="goto(i.component)">
+                    <i :class="i.icon" />
+                    <p>{{ i.name }}</p>
+                </div>
+            </div>
+
+            <div class="column pt3" style="background-color: #edf1f4;">
+                <p v-if="loading">Loading...</p>
+
+                <component v-else :is="type" />
+            </div>
+        </div>
+    </section>
+</template>
+
+<script>
+import TeamsDashboard from './TeamsDashboard'
+import CreateTeam from './CreateTeam'
+import JoinTeam from './JoinTeam'
+import MyTeams from './MyTeams'
+
+export default {
+    name: 'Teams',
+    components: { TeamsDashboard, CreateTeam, JoinTeam, MyTeams },
+    async created ()
+    {
+        this.loading = true;
+
+        await this.$store.dispatch('GET_TEAM_TYPES');
+
+        this.loading = false;
+    },
+    data ()
+    {
+        return {
+            loading: true,
+            items: [
+                { id: 1, name: 'Dashboard', icon: 'fa fa-home teams-icon', component: 'TeamsDashboard' },
+                { id: 2, name: 'Join a Team', icon: 'fa fa-sign-in teams-icon', component: 'JoinTeam' },
+                { id: 3, name: 'Create a Team', icon: 'fa fa-plus teams-icon', component: 'CreateTeam' },
+                { id: 4, name: 'Your Teams', icon: 'fa fa-users teams-icon', component: 'MyTeams' }
+                // sub routes = Team members, Team charts, Team map
+            ]
+        }
+    },
+    computed: {
+
+        /**
+         * What component to show
+         */
+        type ()
+        {
+            return this.$store.state.teams.component_type;
+        }
+    },
+    methods: {
+
+        /**
+         * Change component
+         */
+        goto (type)
+        {
+            this.$store.commit('teamComponent', type);
+        }
+    }
+}
+</script>
+
+<style scoped>
+
+    .fa-users {
+        font-size: 1.75rem !important;
+    }
+
+    .team-flex {
+        display: flex;
+        margin-bottom: 1em;
+        cursor: pointer;
+    }
+
+    .teams-left-col {
+        background-color: #232d3f;
```

Comment:
Should define constant for color.

Reply:
We will when we decide what the colors are! We also plan on migrating to tailwind css and might wait for that to save colour template